qutip/tests/core/data/test_convert.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import pytest
+from scipy import sparse
+from qutip.core.data import to, create, CSR, Dense, dense, csr
```

Comment:
The data layer uses pretty common names - I had generally intended that it would always take qualified access, so `from qutip import data`, then `data.to`, `data.create`, etc, because it's not at all obvious that a function called `to` should be from `qutip` and not any other library a user might have.  We should probably write in that style in the tests, since they are used as a source of usage examples.